pkgs/desktops/gnome-3/3.24/core/gnome-shell/default.nix

```diff
@@ -28,7 +28,7 @@ in stdenv.mkDerivation rec {
       libgweather # not declared at build time, but typelib is needed at runtime
       gnome3.gnome-clocks # schemas needed
       at_spi2_core upower ibus gnome_desktop telepathy_logger gnome3.gnome_settings_daemon
-      pythonEnv gobjectIntrospection (stdenv.lib.getLib dconf) ];
+      pythonEnv gobjectIntrospection dconf ];
```

Comment:
This commit should not be needed anyway since the issue is already fixed in the staging branch.